content/JENKINS/styles/site.css

```diff
@@ -1018,4 +1018,16 @@ body.spaces .aui-tabs > .tabs-menu .menu-item a:visited {
 
 
   #title-heading {clear:left}
-  
\ No newline at end of file
+
+/* Deprecation warning in the header */
+#page {
+  background: url("./assets/jenkenstein.png") no-repeat white;
+  padding-top: 256px;
+}
+
+#page::before {
+  content: "This content is deprecated, you should consult jenkins.io instead ⚠️";
```

Comment:
This is fine, but grammatically it contains a run-on sentence. I also think it might be worth mentioning explicitly that the content may be outdated obsolete to inform users that while the content may still apply to current releases, it also may have been superseded. Optional suggestion for your consideration:

```suggestion
  content: "This content is deprecated and may be have been rendered obsolete by recent releases. ⚠️ For up-to-date documentation, refer to jenkins.io.";
```

Reply:
Better indeed, thanks!
I would omit the dot after jenkins.io though, WDYT?

Reply:
If hyperlinks don't work that seems fine to me.